test/models/v2/message.spec.ts

```diff
@@ -26,20 +26,6 @@ describe('Message model', function() {
     });
   });
 
-  describe('.schemaFormat()', function() {
```

Comment:
Why remove the tests when we still have the functions? 🤔

Reply:
Reverted. good catch.

Reply:
The reality is that message **should not** have the `schemaFormat` method but only the MessageTrait. See https://github.com/asyncapi/parser-api/pull/86/files#diff-ac35ac6e0b0ad3629802a0e04076aa18e94da614f5392e1164dcaacdfe3ee198L157

Reply:
I think i will need to change the way inheritance is being made atm...

Reply:
After thinking again and again, makes sense to keep `schemaFormat` + add `hasSchemaFormat` to `messageObject` as well so we reduce the BC impact. I applied the changes + changes in parser-api. See https://github.com/asyncapi/parser-api/pull/86#issuecomment-1651893775

Reply:
So this test method is back now